Simplified #content .primary p:nth-child selector. closes #1450

webinista opened the pull request:
- Replacing #content .primary p:nth-child(1) with .primary p:nth-child(q)
- Adds .post p:nth-child(1) to avoid visual regression on blog.
- Reformats some blocks for easier readability.

modified:   _sass/_va.scss

webinista committed:
Simplified #content .primary p:nth-child selector. closes #1450
- Replacing #content .primary p:nth-child(1) with .primary p:nth-child(q)
- Adds .post p:nth-child(1) to avoid visual regression on blog.
- Reformats some blocks for easier readability.

modified:   _sass/_va.scss

webinista committed:
Simplified #content .primary p:nth-child selector. closes #1450
- Replacing #content .primary p:nth-child(1) with .primary p:nth-child(q)
- Adds .post p:nth-child(1) to avoid visual regression on blog.
- Reformats some blocks for easier readability.

modified:   _sass/_va.scss

crwallace commented on _sass/_va.scss:
```diff
@@ -102,8 +100,8 @@ a {
 a:visited {
-  color: #4c2c92
-  }
+  color: #4c2c92;
```
Can we change this to `$color-visited` since it is already in `_variables.scss`?

crwallace commented on _sass/_va.scss:
```diff
@@ -1154,7 +1153,15 @@ figure {
-  a, p, li, h1, h2, h3, h4, h5, h6, dl, dd, dt, caption, figure, blockquote {color: #444; line-height: 1.65em;}
+  a, p, p:nth-child(1), li, h1, h2, h3, h4, h5, h6, dl, dd, dt, caption, figure,   
+  blockquote {
+    color: #444; 
```
Do we only use this color once or should we make this a variable?

crwallace commented on _sass/_va.scss:
```diff
@@ -387,13 +393,6 @@ h6 {font-size: 1.15em; font-weight: bold;}
     font-size: 1em;
```
Do we only use this color once or should we make this a variable?

crwallace commented:
Couple of comments but otherwise LGTM!

webinista committed:
Merge branch 'master' into issue#1450b

webinista committed:
Merge branch 'master' into issue#1450b

webinista committed:
Merge branch 'issue#1450b' of github.com:department-of-veterans-affairs/vets-website into issue#1450b

webinista committed:
Changing colors to variables.

- Changing #444 to $color-gray-cool-dark.
- Changing a:visited color to $color-visited.

webinista commented:
Can you re-review just as a sanity check? If it looks good and the branch is still up-to-date, please merge.

crwallace commented:
LGTM!
